scrapper.py

```diff
@@ -86,6 +113,8 @@ def validate_config(crawler_path):
     pass
 
 
+seed_urls, max_articles, max_articles_per_seed = validate_config(CRAWLER_CONFIG_PATH)
+
```

Comment:
print your links here and check if they are correct - try to open some of them - if correct - so move on